cachecontrol/caches/file_cache.py

```diff
@@ -5,18 +5,24 @@
 import hashlib
 import os
 from textwrap import dedent
+from typing import TYPE_CHECKING, Optional, Type
 
 from ..cache import BaseCache, SeparateBodyBaseCache
 from ..controller import CacheController
 
+if TYPE_CHECKING:
+    from io import BufferedIOBase, BufferedWriter
+    from filelock import BaseFileLock
+
+
 try:
     FileNotFoundError
 except NameError:
     # py2.X
     FileNotFoundError = (IOError, OSError)
 
 
-def _secure_open_write(filename, fmode):
+def _secure_open_write(filename: str, fmode: int) -> "BufferedWriter":
```

Comment:
```suggestion
def _secure_open_write(filename: str, fmode: int) -> "IO[bytes]":
```
They work the same in typing, but `IO[bytes]` is simpler